Exercise1/myFile-rt/src/com/tridiumuniversity/myFile/BFile.java

```diff
@@ -210,13 +213,15 @@ public class BFile extends BComponent {
 
   public void doTouch()
   {
-    System.out.println("Hello, World  print");
     setModifiedDate((BAbsTime.now()));
   }
   public void doPrint()
   {
-    System.out.println("Hello, World! 2nd Component1");
-//    setTimeExecuted(BAbsTime.now());
+    System.out.println("Filename: " + getFilename());
   }
 
+  private String getFilename()
```

Comment:
Since I was getting error in line 220 at getFilename() I have created this return method, not sure it fits, please suggest if any.

Reply:
I suspect you'd get an error when calling `getFilename()` because the name of the property is now `fileName`, not `filename`. This means the getter method is now `getFileName()`, not `getFilename()`. You shouldn't ever need to define your own method like this to get a property. You should always use the method generated by slotomatic.

Reply:
Yes, now its corrected.